content/en/docs/concepts/workloads/controllers/ttlafterfinished.md

```diff
@@ -0,0 +1,80 @@
+---
+reviewers:
+- janetkuo
+title: TTL Controller for Finished Resources
+content_template: templates/concept
+weight: ??
+---
+
+{{% capture overview %}}
+
+The TTL controller provides a TTL mechanism to limit the lifetime of resource
+objects that have finished execution. Currently, TTL controller only handles
+[Jobs](/docs/concepts/workloads/controllers/jobs-run-to-completion/) for
+now, and may be expanded to handle other resources that will finish execution,
+such as Pods and custom resources.
+
+Alpha Disclaimer: this feature is currently alpha, and can be enabled with
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+`TTLAfterFinished`.
+
+
+{{% /capture %}}
+
+
+{{< toc >}}
+
+
+{{% capture body %}}
+
+## TTL Controller
+
+The TTL controller only supports Jobs for now. You can use this feature to clean
+up finished Jobs (either `Complete` or `Failed`) automatically by specifying the
+`.spec.ttlSecondsAfterFinished` field of a Job, 
+see [example](/docs/concepts/workloads/controllers/jobs-run-to-completion/#clean-up-finished-jobs-automatically). 
+The TTL controller will assume that a resource is eligible to be cleaned up
+TTL seconds after the resource has finished, i.e. TTL has expired. When the
+resource is deleted, its lifecycle guarantees, such as finalizers, will be
+honored.
+
+The TTL seconds can be set at any time -- for example, you can specify it in the
+resource manifest, set it at resource creation time, or set it after the
+resource has finished. You can also use
+[mutating admission webhooks](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#admission-webhooks)
+to set this field dynamically.
+
+In the future, we plan to expand TTL controller to other resources that will
```

Comment:
We usually try to avoid making [promises about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future) in user docs.

Reply:
Removed this sentence.